src/dhcp/aca_dhcp_state_handler.cpp

```diff
@@ -80,8 +76,28 @@ int Aca_Dhcp_State_Handler::update_dhcp_state_workitem(const DHCPState current_D
       break;
     }
   }
-
-  switch (current_DhcpState.operation_type()) {
+  alcor::schema::OperationType current_operation_type =
```

Comment:
since this is a temporary fix :), it will be great to add a bunch of comment on why we are doing it, what does it do and when we should remove it.

Reply:
Comment added, thanks for your suggestions 👍